src/main/java/org/folio/marc/migrations/client/InstanceStorageClient.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.marc.migrations.client;
+
+import org.folio.marc.migrations.domain.dto.BulkRequest;
+import org.folio.marc.migrations.domain.dto.BulkResponse;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@FeignClient("instance-storage")
+public interface InstanceStorageClient {
```

Comment:
I suggest to have one client 
You can use logic similar to https://github.com/folio-org/mod-search/blob/master/src/main/java/org/folio/search/client/InventoryReferenceDataClient.java
Create an enum with URLs and have mapper to this enum from EntityType. 
Then we will have cleaner BulkStorageService

Reply:
Done, please review